esmvaltool/references/woa.bibtex

```diff
@@ -0,0 +1,3 @@
+@article{woa,
```

Comment:
Split the fields.

The is another reference to Locarini, possibly a duplicate.

Reply:
the year is different: woa:2013
but locarini: 2010